packages/wrangler/src/pages.tsx

```diff
@@ -1236,6 +1244,11 @@ const createDeployment: CommandModule<
             await new Promise((resolve) =>
               setTimeout(resolve, attempts++ * 1000)
             );
+
+            if ((e as { code: number }).code === 8000013) {
+              // Looks like the JWT expired, fetch another one
+              await refreshJwt();
```

Comment:
Functions with side-effects like this feel a bit magical. Why not just assign the `jwt` to a return value?

```suggestion
              jwt = await getUploadToken();
```